components/Button.js

```diff
@@ -14,8 +14,8 @@ const sx = {
   cursor: 'pointer'
 }
 
-const Button = ({ ...props }) => (
-  <button {...props} style={sx} />
+const Button = ({ children, ...props }) => (
+  <button {...props} style={sx} >{children}</button>
 )
```

Comment:
Passing children explicitly inside the element was necessary for Inferno. Apparently, all other libraries handle this syntax.